pkg/gatewayserver/io/mqtt/mqtt_test.go

```diff
@@ -93,8 +93,7 @@ func TestAuthentication(t *testing.T) {
 					client.Disconnect(uint(timeout / time.Millisecond))
 				}
 			} else {
-				a.So(ok, should.BeTrue)
```

Comment:
Why did this change?

Reply:
This is a breaking change in Paho.

Formerly, on timeout, `true` was returned.

Now, the return value indicates whether the action was successful (i.e. `false` on timeout). So in this statement, we assert `false` as we expect a timeout when the test case is supposed to fail.

See https://github.com/eclipse/paho.mqtt.golang/commit/ba971f185f1bce82b0e8d19c74da95e37c68234a#diff-ac704b2e102a5731d9d1b1707d3bc7a1R80